javalib/src/main/scala/java/net/StandardSocketOptions.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 Arman Bilge
```

Comment:
Thank you, but the copyright notice really isn't necessary 😂 I'm happy to license the code directly to the Scala Native project. It is rather straightforward anyway, so attempting to lay claim would be embarrassing 😉